src/controller/python/chip/clusters/Attribute.py

```diff
@@ -503,6 +503,26 @@ def OverrideLivenessTimeoutMs(self, timeoutMs: int):
             lambda: handle.pychip_ReadClient_OverrideLivenessTimeout(self._readTransaction._pReadClient, timeoutMs)
         )
 
+    def GetReportingIntervals(self) -> Tuple[int, int]:
```

Comment:
Should/could we name this with some suffix like `GetReportingIntervalsSeconds` ?

The C++ header has the argument named `seconds` but that would be lost to python.

Reply:
OK